openedx/frontend-app-course-authoring
# feat: add file size and usage metrics

This PR adds the file size and usage metrics to the asset info modal.

Testing

1. Navigate to the course authoring assets page
2. Click on the vertical three dots on an assets and select info
3. Check that the asset's file size section is populated and has the expected file size
4. Close the modal
5. Add a new file
6. Open the new file's info modal
7. Check that the asset's file size section is populated and has the expected file size

This PR is dependent on

- edx-platform [PR #33068](https://github.com/openedx/edx-platform/pull/33068)
